dotCMS/src/integration-test/java/com/dotmarketing/filters/FiltersTest.java

```diff
@@ -163,120 +164,90 @@ public String answer(InvocationOnMock invocation) throws Throwable {
     }
 
     @Test
-    public void shouldWorkVanityUrl() throws IOException, DotDataException {
+    public void shouldWorkVanityUrl() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)